index.html

```diff
@@ -51,6 +51,114 @@ <h1 class="my-name" >I'm Omotoyosi. <br>
                 </div>
             </div>
         </section>
-    </div>
+    </section>
+    <section class="second-section">
+        <div class="item works-1">
+            <img src="images/works-1.svg" alt="template-image" srcset="">
+            <p class="title-1">Tonic</p>
+            <ul class="track">
+                <li class="live">CANOPY</li>
+                <li><img src="images/Counter.svg" alt="counter" srcset=""></li>
+                <li>Back End Dev</li>
+                <li><img src="images/Counter.svg" alt="" srcset=""></li>
+                <li>2015</li>
+            </ul>
+            <p class="some-text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="stacks">
+                <li> <button class="stack-button"><a href="http://"> HTML</a></button></li>
+                <li><button class="stack-button"><a href="http://"> CSS</a></button></li>
+                <li><button class="stack-button"><a href="http://">JavaScript</a></button></li>
+            </ul>
+            <button class="see-proj" ><a class="project-link" href="">See project</a></button>
```

Comment:
- [ ] The button is great but the background color is different from the Figma template. Kindly use the correct white background color to match the template for each project card button.